docs/features/colorbutton/README.md

```diff
@@ -72,6 +72,30 @@ CKEditor will then output the color definition as `<font>` elements with `color`
 	<p><font color="#800080">This is my text color.</font><br/>
 	<font style="background-color:#FFFF00;">This is my background color</font></p>
 
+Since CKEditor 4.15.0 it is possible to use `class` instead of `style` to set more advanced styles to text.
+
+You need to set {@linkapi CKEDITOR.config.colorButton_colors custom color names}.
```

Comment:
You can use `colorName` despite changing `colorButton_colors` but then it will use [default color](https://github.com/ckeditor/ckeditor4/blob/de96c26164b0d34799df800b88c923a72fd595ba/plugins/colorbutton/plugin.js#L483) codes as names. Setting `colorButton_colors` to get friendly names/classes is a good tip but I would move it to the end of this section - maybe with some example too?